docs/source/tools_integration/pyspark.md

```diff
@@ -19,7 +19,7 @@ Optimal configuration for Spark depends on the setup of your Spark cluster.
 
 Before any `PySpark` operations are performed, you should initialise your [`SparkSession`](https://spark.apache.org/docs/latest/sql-getting-started.html#starting-point-sparksession) in your custom project context class, which is the entrypoint for your Kedro project. This ensures that a `SparkSession` has been initialised before the Kedro pipeline is run.
 
-Below is an example implementation to initialise the `SparkSession` in `<project-name>/src/<python_package>/custom_context.py` by reading configuration from the `spark.yml` configuration file created in the previous section:
+Below is an example implementation to initialise the `SparkSession` in `<project-name>/src/<package_name>/custom_context.py` by reading configuration from the `spark.yml` configuration file created in the previous section:
```

Comment:
```suggestion
Below is an example implementation to initialise the `SparkSession` in `src/<package_name>/custom_context.py` by reading configuration from the `spark.yml` configuration file created in the previous section:
```

Reply:
Changes made in [344a6fb](https://github.com/kedro-org/kedro/pull/1753/commits/344a6fb49d232371b8f592ee5ccc35d681bee043).